torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14478,6 +14478,7 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                     supports_out=False,
                     supports_forward_ad=True,
                     supports_fwgrad_bwgrad=True,
+                    supports_one_python_scalar=True,
```

Comment:
Why was this set to false previously? Was that an oversight?

Reply:
My understanding is that some of the tests were un-happy with the tensor created on the fly there and were failing.